### PR TITLE
feat: add workflow to build and push container

### PR DESCRIPTION
**Pull request by kaitoz11**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    - name: Login to Docker Hub
+      uses: docker/login-action@v2
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+    - name: Ship to Docker Hub
+      uses: docker/build-push-action@v4
+      with:
+        contex: .
```
Comment (ChienNQuang) [inline .github/workflows/build.yml:28]: contex or context?

Comment (kaitoz11) [reply .github/workflows/build.yml:28]: ohhhh fuk

**Review by StarryFolf — APPROVED** [review]

**Review by Vzart — APPROVED** [review]
Nice typo btw

**Comment by kaitoz11** [comment]
@ChienNQuang Is there any problem?

**Review by ChienNQuang — APPROVED** [review]

**Pull request merged**